client/conn.go

```diff
@@ -41,6 +41,7 @@ var (
 	connectionID uint64
 	seqNo        uint64
 	randSource   = rand.New(rand.NewSource(time.Now().UnixNano()))
+	GrandLock    sync.Mutex
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported var GrandLock should have comment or be unexported